src/functions/write_detrend_data.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
+
+import pandas as pd
+
+sys.path.append("./src/functions")
+from detrend import LinearMA  # noqa: E402
+
+# SPECIFY DETREND MODEL AND DIRECTORY IN WHICH SAVE PROCESSED DATA
```

Comment:
nit: no uppercase, write in lower case (or lowercase with first letter uppercased)

Reply:
done